### ISSUE TITLE
SQL injection vulnerabilities

### ISSUE THREAD

**Issue by joshkel** [ag-grid/ag-grid-server-side-nodejs-example#33]
I noticed that this repo makes no attempt to validate inputs or escape its SQL. I realize that it's just an example, but, since it describes itself as a reference implementation and is recommended from https://www.ag-grid.com/react-data-grid/server-side-operations-nodejs/ with no caveats or warnings, it seems to me that it should either properly handle its SQL or include highly visible warnings indicating these limitations.

**Comment by miszczu** [comment]
I agree that it should be pointed out as a warning in both ag-grid documentation page and in this repo's README file
